publisher/test/178.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I don't see this new file path loaded by any of the lines in your diff. Is this necessary?

Reply:
I reference it on line 138 of publisher_test.go. I wanted to make sure publisher has a copy of the EE if I extend this test. It can probably go.

Reply:
Yeah, I'd delete the file, since none of the tests actually load it. We don't need more copies of 178.pem floating around, it's in a bunch of places already and needs to go away.